src/util/netif-mgmt.c

```diff
@@ -73,6 +73,23 @@
 #define SIOCSIFLLADDR SIOCSIFHWADDR
 #endif
 
+// incase strlcpy is not found in libc
+size_t _my_strlcpy(char *aDst, const char *aSrc, size_t aSize) 
```

Comment:
Not sure if we would need this since `wpantund` provides/uses its own `strlcpy` (if not available) from `src/missing/strlcpy` file.
Check this source https://github.com/openthread/wpantund/blob/master/src/missing/strlcpy/strlcpy.c#L30